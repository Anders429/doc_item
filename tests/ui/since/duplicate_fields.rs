use doc_item::since;

#[since(content="content 1", content="content 2")]
fn duplicate_fields() {}

fn main() {}
