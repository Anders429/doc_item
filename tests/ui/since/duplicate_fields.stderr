error: Duplicate field `content`
 --> $DIR/duplicate_fields.rs:3:30
  |
3 | #[since(content="content 1", content="content 2")]
  |                              ^^^^^^^^^^^^^^^^^^^
